webportal/webapp/src/services/cognito.service.ts

```diff
@@ -28,33 +29,24 @@ export interface Callback {
 
 @Injectable()
 export class CognitoService {
-   
+    constructor(@Inject(WindowToken) private window: Window) { }
+
+
     public static _REGION = environment.REGION // User pool AWS region
     public static _USER_POOL_ID = environment.USER_POOL_ID // User pool ID
     public static _CLIENT_ID = environment.CLIENT_ID // App client ID
     public static _IDENTITY_PROVIDER = environment.IDENTITY_PROVIDER // User pool Identity provider name
     public static _APP_DOMAIN = environment.APP_DOMAIN // App domain name
     public static _IDP_ENDPOINT = "cognito-idp." + CognitoService._REGION + ".amazonaws.com/" + CognitoService._USER_POOL_ID
-    public static _REDIRECT_URL = environment.REDIRECT_URL // Re-direct URL for the user pool
     
     public static _POOL_DATA:any = {
         UserPoolId: CognitoService._USER_POOL_ID,
         ClientId: CognitoService._CLIENT_ID
     };
-    public static _AUTH_DATA = {
```

Comment:
This got pulled in from master... this had to get refactored slightly to use the injected window.